crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -3,9 +3,22 @@ type Address implements ObjectOwner {
 	Similar behavior to the `transactionBlockConnection` in Query but
 	supports additional `AddressTransactionBlockRelationship` filter
 	"""
-	transactionBlockConnection(first: Int, after: String, last: Int, before: String, relation: AddressTransactionBlockRelationship, filter: TransactionBlockFilter): TransactionBlockConnection
+	transactionBlockConnection(
```

Comment:
This is just applying the prettier formatting, I don't think we are running a lint step in CI for this, but anyone who edits this with prettier set up in the IDE will apply this formatting.  Can revert if needed

Reply:
actually, if this is generated from our implementation, this should probably be reverted, but it may be worth running prettier on it as part of whatever generates this

Reply:
Correct, this is generated when the test runs!